raiden/channel.py

```diff
@@ -410,6 +410,7 @@ def settled_block(self):
         return self._settled_block
 
     def set_opened(self, block_number):
+        # TODO: ensure the same callback logic as in set_settled
```

Comment:
No, but I did run into the `RuntimeError` in `set_settled` (which is there to prevent duplicate callbacks). It is a pretty defensive safe-guard and I believe the removal of https://github.com/raiden-network/raiden/pull/391/files/99ba6e5066694af9f6bd0a025c778a1b86626106#diff-08685f521533606c5888755314d415efL618 did fix `RuntimeError` I encountered.